packages/react-atlas-core/src/Dropdown/Dropdown.js

```diff
@@ -276,22 +257,22 @@ class Dropdown extends React.PureComponent {
             }
           </div>
         }
-        <div onClick={(e) => { this._toggle("click", e); }} styleName={cx("content")}>
+        <div onClick={(e) => { this._toggle("click", e); }} styleName={"content"}>
           <div style={{ "width": buttonWidth + "px" }}>
             <ButtonCore
-              styleName={cx("buttonClass")}
+              styleName={"buttonClass"}
               className={dropdownButtonClasses}
             >
               <span>{this.state.output}</span><i styleName="arrow" />
             </ButtonCore>
           </div>
           {this.state.active &&
-            <span styleName={"list"}>{bound_children}</span>
+            <ul styleName={"list"}>{bound_children}</ul>
```

Comment:
Is UL the best thing to use here?  Because it technically means <unorder list> which is usually accompanied by a list of items like 
<ul>
  <li></li>
  <li></li>
</ul>

so this was why I originally changed it to span.